oonib/test/test_report.py

```diff
@@ -206,5 +187,3 @@ def test_create_update_and_close_report(self):
             for i in range(report_entry_count):
                 self.assertEqual(yaml.safe_load(sample_report_entry),
                                  written_report.next())
-
-        self.directories.add('ZZ')
```

Comment:
Wouldn't it more accurate to add a test case in which you update a report by pushing a json entry?